app/src/main/java/com/infomaniak/drive/utils/Extensions.kt

```diff
@@ -508,8 +508,10 @@ fun Fragment.observeAndDisplayNetworkAvailability(
                 TransitionManager.beginDelayedTransition(noNetworkBindingDirectParent, this)
             }
 
-            noNetworkBinding.noNetwork.isGone = isNetworkAvailable == null || isNetworkAvailable == true
-            additionalChanges?.invoke(isNetworkAvailable == true)
+            (isNetworkAvailable != false).let { hasNetwork ->
```

Comment:
either create a variable like in the other place or directly use 2 times isNetworkAvailable != false